q2_gneiss/plot/_plot.py

```diff
@@ -134,8 +134,29 @@ def f(x):
             sample_palette = pd.Series(
                 sns.color_palette("Set2", len(c.value_counts())),
                 index=c.value_counts().index)
+
```

Comment:
Maybe we could simplify this a bit using a `try/except/else`?

```python
try:
    pd.to_numeric(metadata.to_series())
except ValueError:
    pass
else:
    raise ValueError('...')
```

Reply:
For sure, I like that solution a lot